verification/src/http_server/handlers/verification/solidity/types.rs

```diff
@@ -13,23 +18,41 @@ pub struct VerificationRequest<T> {
 }
 
 #[derive(Debug, Serialize, PartialEq)]
+#[serde(rename_all = "camelCase")]
 pub struct VerificationResponse {
     pub verified: bool,
 }
 
-#[derive(Debug, Deserialize, PartialEq)]
-#[serde(rename_all = "camelCase")]
-enum EvmVersion {
-    Homestead,
-    TangerineWhistle,
-    SpuriousDragon,
-    Byzantium,
-    Constantinople,
-    Petersburg,
-    Istanbul,
-    Berlin,
-    London,
-    Default,
+#[derive(Debug, Serialize, Deserialize, PartialEq)]
+struct EvmVersion(
+    #[serde(serialize_with = "use_display", deserialize_with = "use_from_str")]
+    ethers_solc::EvmVersion,
+);
+
+fn use_from_str<'de, D: serde::Deserializer<'de>, T: FromStr<Err = String>>(
+    deserializer: D,
+) -> Result<T, D::Error> {
+    let s: &str = serde::de::Deserialize::deserialize(deserializer)?;
+    T::from_str(s).map_err(serde::de::Error::custom)
+}
+
+fn use_display<T: std::fmt::Display, S: serde::Serializer>(
+    value: &T,
+    serializer: S,
+) -> Result<S::Ok, S::Error> {
+    serializer.collect_str(value)
+}
+
+impl From<ethers_solc::EvmVersion> for EvmVersion {
+    fn from(v: ethers_solc::EvmVersion) -> Self {
+        Self(v)
+    }
+}
+
+impl From<EvmVersion> for ethers_solc::EvmVersion {
+    fn from(v: EvmVersion) -> Self {
+        v.0
+    }
 }
```

Comment:
I've implemented from traits for future use, like serialization

If we implement our own enum, what happens when the new evm version comes? Right now we only need to update the library, our code will be intact, but if we'll implement our own enum, we'll need to update it too

The most flexible variant is just to accept the string, and pass it to the compiler, because we basically don't care if it's valid (also some newer versions aren't supported on old compilers), but this isn't compatible with ethers_solc library as of now